src/new-client/src/app/lib/types/index.tsx

```diff
@@ -41,6 +41,16 @@ type ApplyAvailabilityTemplateRequest = {
   template: AvailabilityTemplate;
 };
 
+type AvailabilityCreatedEvent = {
```

Comment:
Yeah, I had exactly the same thought to be honest. I almost made two different interfaces in the frontend, but then thought "Well why don't we make that split on the backend too?" - and the answer was because they both share a document type for speed of querying. 

Is it alright if we leave it as-is for now, but maybe discuss & improve in future?

Reply:
Yes, I think this will probably resolve itself if we add more audit events and if we don't then this will work just fine